integration/stability/agent_stability_test.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+set -e -x
```

Comment:
Can you add a short paragraph for describing what is the purpose of this test?
It would be useful to not confuse users trying to run the test, as they may notice that the `timeout` is huge.